cli/src/cli.rs

```diff
@@ -679,26 +672,6 @@ pub fn parse_command(
         }
         ("upgrade-nonce-account", Some(matches)) => parse_upgrade_nonce_account(matches),
         // Program Deployment
-        ("deploy", Some(matches)) => {
```

Comment:
What do you think about leaving the `Subcommand` and using this match case to return an [Error exit](https://docs.rs/clap/2.34.0/clap/struct.Error.html#method.exit) pointing users at `solana program deploy`?

Reply:
Might be good for user experience. I've added back deploy in 2a5b9101fd7da8d3377f8454dff0af3764bc217f